src/plugins/orangehrmAdminPlugin/Dao/LocalizationDao.php

```diff
@@ -129,7 +129,10 @@ private function getTranslationsQueryBuilderWrapper(
 
         if (!is_null($i18NTargetLangStringSearchFilterParams->getOnlyTranslated())) {
             if ($i18NTargetLangStringSearchFilterParams->getOnlyTranslated() === true) {
+                $q->andWhere($q->expr()->isNotNull('translation.value'));
                 $q->andWhere($q->expr()->isNotNull('translation.translated'));
+                $q->andWhere($q->expr()->notIn('translation.value', ':translated'));
+                $q->setParameter('translated', '');
```

Comment:
```suggestion
```
I want to discuss this

Reply:
Add these lines after BA review to avoid getting empty values for the translated strings. But a there can be removed translated strings.